src/components/tab-group/tab-panel/bl-tab-panel.ts

```diff
@@ -40,7 +40,7 @@ export default class BlTabPanel extends LitElement {
    * This attribute set by `tab-group` to make panel visible or hidden.
    */
   @property({ type: Boolean, reflect: true })
-  visible = false;
+  hidden = true;
```

Comment:
I just realized that, setting default value of  a boolean attribute as `true` is a bit problemmatic. Because it's not technically possible to set this to false via HTML.

```html
<bl-tab-panel hidden="false">
```

doesn't work.  On the other hand, this property is supposed to set by `BlTabGroup` component only. I'm not sure if default value should be `true` or `false`.

Reply:
As discussed, changed this to a state instead of a property.